### PR TITLE
{tools}[foss/2022a] Sniffles v2.0.7 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
Test report by @PetrKralCZ
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4008.donphan.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 530.30.02, Python 3.6.8
See https://gist.github.com/PetrKralCZ/f7367e38d3e79cbb1c63c7aa47a453b5 for a full test report.

File: easybuild/easyconfigs/s/Sniffles/Sniffles-2.0.7-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    ('Python', '3.10.4'),
+    ('Pysam', '0.20.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    (name, version, {
+        'modulename': '%(namelower)s',
```
Comment (boegel) [inline easybuild/easyconfigs/s/Sniffles/Sniffles-2.0.7-foss-2022a.eb:22]: This shouldn't be needed, using the lowercase software name for the `import` check is the default

File: easybuild/easyconfigs/s/Sniffles/Sniffles-2.0.7-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        'sources': [SOURCELOWER_TAR_GZ],
+        'checksums': ['bf532eb0b3aaeaee840100a3bb3339f1d5175ba98adfe3cf19685b0abb0885a1'],
+    }),
+]
+
+sanity_check_commands = [
+    "python -c 'from sniffles import sv'",
+    "sniffles --help",
+]
```
Comment (boegel) [inline easybuild/easyconfigs/s/Sniffles/Sniffles-2.0.7-foss-2022a.eb:31]: If there's a custom binary, then also define `sanity_check_paths` to overrule the default which only checks for the `site-packages` dir:

```suggestion
sanity_check_paths = {
    'files': ['bin/sniffles'],
    'dirs': ['lib/python%(pyshortver)s/site-packages'],
}

sanity_check_commands = [
    "python -c 'from sniffles import sv'",
    "sniffles --help",
]

File: easybuild/easyconfigs/s/Sniffles/Sniffles-2.0.7-foss-2022a.eb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+moduleclass = 'tools'
```
Comment (boegel) [inline easybuild/easyconfigs/s/Sniffles/Sniffles-2.0.7-foss-2022a.eb:33]: `bio` is probably better?

**Review by boegel — DISMISSED** [review]

**Comment by PetrKralCZ** [comment]
Test report by @PetrKralCZ
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4008.donphan.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 530.30.02, Python 3.6.8
See https://gist.github.com/PetrKralCZ/d801bd2a6a4a01e522a081f69d1b53a8 for a full test report.

**Comment by PetrKralCZ** [comment]
Test report by @PetrKralCZ
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4008.donphan.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 530.30.02, Python 3.6.8
See https://gist.github.com/PetrKralCZ/188cb7e21ed904b2d4c349534bcf18b5 for a full test report.

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
tcn331.local.snellius.surf.nl - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/d679643955798839235bb9617c7c668c for a full test report.

File: easybuild/easyconfigs/s/Sniffles/Sniffles-2.0.7-foss-2022a.eb [hunk 4]
```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonBundle'
+
+name = 'Sniffles'
+version = '2.0.7'
+
+homepage = 'https://github.com/fritzsedlazeck/Sniffles'
+description = """A fast structural variant caller for long-read sequencing,
+ Sniffles2 accurately detect SVs on germline, somatic and population-level for PacBio and Oxford Nanopore read data."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
```
Comment (casparvl) [inline easybuild/easyconfigs/s/Sniffles/Sniffles-2.0.7-foss-2022a.eb:10]: Both dependencies are at GCCcore/GCC level, does Sniffles really need `foss`? Which components of the foss toolchain does it use?

**Comment by PetrKralCZ** [comment]
Test report by @PetrKralCZ
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4004.donphan.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 530.30.02, Python 3.6.8
See https://gist.github.com/PetrKralCZ/f3ecef4efc90cdd40392b2d31023f333 for a full test report.

**Comment by casparvl** [comment]
Test report by @casparvl
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
tcn399.local.snellius.surf.nl - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/4cc26517bcec81ff25130ad00b2b55db for a full test report.

**Review by casparvl — APPROVED** [review]
Lgtm

**Comment by casparvl** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**